guides/micronaut-aws-secretsmanager-rotation/micronaut-aws-secretsmanager-rotation.adoc

```diff
@@ -0,0 +1,169 @@
+= @guideTitle@
+
+@guideIntro@
+
+Authors: @authors@
+
+Micronaut Version: @micronaut@
+
+include::{commondir}/common-gettingStarted.adoc[]
+
+include::{commondir}/common-requirements.adoc[]
+
+include::{commondir}/common-completesolution.adoc[]
+
+== Writing the App
+
+include::{commondir}/common-cli-or-launch.adoc[]
+
+[source,bash]
+----
+mn create-function-app example.micronaut.micronautguide --features=aws-lambda --build=@build@ --lang=@lang@
+----
+
+include::{commondir}/common-build-lang-arguments.adoc[]
+
+If you use https://launch.micronaut.io[Micronaut Launch], select serverless function as application type and add `aws-lambda` feature.
+
+include::{commondir}/common-default-package.adoc[]
+
+include::{commondir}/common-annotationprocessors.adoc[]
+
+include::{commondir}/common-function-delete-sample-code.adoc[]
+
+== Code
```

Comment:
IMO this guide needs some explanation of what it is trying to achieve, it jumps straight into the code without defining "why" and "what" the reader is doing.

Reply:
This is the intro: 

> Learn how to create an AWS Lambda function with the Micronaut Framework to rotate a secret stored in AWS Secrets Manager

Reply:
It is also a companion guide to https://guides.micronaut.io/latest/micronaut-security-keys-jwks-gradle-java.html and https://guides.micronaut.io/latest/micronaut-aws-secretsmanager.html